cookiecutter.json

```diff
@@ -1,5 +1,6 @@
 {
     "project_name": "your-project-with-dashes",
     "package_name": "{{ cookiecutter.project_name.lower().replace('-', '_') }}",
-    "project_short_description": "A short one-line description of the project"
+    "project_short_description": "A short one-line description of the project",
+    "version": "0.1.dev0"
```

Comment:
Hoe minder gevraagd, hoe prettiger (het lijstje was vroeger twee keer zo lang).
Dus: 0.1 is helemaal prima als eerste versie zonder het te vragen. Je kan het altijd aanpassen in de 0.1% van de projecten waar je niet met 0.1 begint.